### ISSUE TITLE
Cannot get get_kbrd() to return typed character

### ISSUE THREAD

**Issue by AstroArtie** [gnudatalanguage/gdl#1230]
If I do:
`GDL> print," a letter: "& wait, 3& s = get_kbrd(0)& print, byte(s)`
and type a letter at the prompt within the 3seconds, it prints '0' for '`byte(s)`' and then it puts my letter at the new prompt, e.g.:
`GDL> a`
Same behavior no matter what argument, or no argument, I give `get_kbrd()`.
I am on a laptop running Fedora Core 34 (5.15.7-100.fc34.x86_64), with readline-8.1-2.fc34.x86_64 installed (but also compat-readline5-5.2-39.fc34.x86_64).
I am not sure what info is relevant for this issue.
I am new to GDL but a long-time IDL'ist :-)
Any help will be most appreciated.

**Comment by AstroArtie** [comment]
This problem appears only after having issued a '`get_kbrd(0)`' command.
Then all subsequent get_kbrd calls seems to assume a 0 argument.

**Comment by GillesDuvert** [comment]
Indeed, I can reproduce: using once get_kbrd(0) will remove the possibilty of using a wait time as in get_kbrd(3) . Nice bug.

**Comment by AstroArtie** [comment]
Good to hear that I'm not going crazy ;-)

**Comment by GillesDuvert** [comment]
looks like a regression somewhere after 0.9.8 ...

**Comment by GillesDuvert** [comment]
Cannot reproduce as of today, windows, mac, linux...
Will close, please open a new issue if needed
